source/voice_control/best_practices.markdown

```diff
@@ -0,0 +1,45 @@
+---
+title: Best practices with Assist
+related:
+  - docs: /voice_control/android
+    title: Assist on Android devices
+  - docs: /voice_control/apple
+    title: Assist on Apple devices
+  - docs: /voice_control/thirteen-usd-voice-remote/
+    title: Build a 13$ voice remote using an ESPHome device
+  - docs: /voice_control/builtin_sentences
+    title: Sentences starter kit
+  - url: https://www.nabucasa.com/config/
+    title: Home Assistant Cloud
+---
+
+Using Assist consists of saying supported commands while targeting exposed devices and entities.
+
+Let's go through these two key concepts
+
+You control what data Assist has access to, and what it can control.
+Every entity in Home Assistant can be exposed or not to Assist.
+Learn how in [Exposing your entities to Assist](https://www.home-assistant.io/voice_control/voice_remote_expose_devices/) 
+
+On top of exposing the needed data, it is worth noting that you will most likely target entities through areas and floors, like:
+
+- *Turn off the office lights*
+
+So make sure your devices and entities are correctly assigned to areas, and your areas are correctly assigned to floors.
+Learn how [here](https://www.home-assistant.io/voice_control/assign_areas_floors/).
```

Comment:
```suggestion
Learn how [here](/voice_control/assign_areas_floors/).
```

Reply:
Not resolved yet, there are a lot more places where the full url is used and should be changed.

Reply:
you are totally right, fixing it